opsdroid/const.py

```diff
@@ -25,6 +26,7 @@
     "configuration/example_configuration.yaml",
 )
 REGEX_PARSE_SCORE_FACTOR = 0.6
+MODULE_ROOT = os.path.dirname(os.path.abspath(opsdroid.__file__))
```

Comment:
You could shift this higher up (I suggest line 8) and then use this in the definition of `LOCALE_DIR`.